contrib/twirllib/src/TwirlModule.scala

```diff
@@ -25,10 +25,10 @@ trait TwirlModule extends mill.Module {
         coursier.LocalRepositories.ivy2Local,
         MavenRepository("https://repo1.maven.org/maven2")
       ),
-      Lib.depToDependency(_, "2.12.5"),
+      Lib.depToDependency(_, "2.13.8"),
```

Comment:
Is the twirl compiler now on Scala 2.13?

And even if so, we may want to keep support of older versions, at least in Mill 0.10.x

Alternatively, we may state some (non-)compatibility notes in the plugin documentation page and the release notes.

Reply:
2.13 support since twirl 1.4.2 https://mvnrepository.com/artifact/com.typesafe.play/twirl-compiler

Does this line of code mean it's either 2.12 or 2.13 support? I don't know much about the whole "binary incompatibility" situation. Is it a lot more complex to support older versions too?

Reply:
In this specific use case, it means, all Scala ivy deps (those with a double colon before the name) will resolve to the given Scala version. You can't mix them in the same classpath, so yes, it's either 2.12 or 2.13.

We can support both versions and we should, but it may not work with this hardcoded version here.